### DISCUSSION TITLE
Safari Burps

### DISCUSSION THREAD

**Discussion by ghost** [usnistgov/macos_security#254] [Q&A]
I don't know what else to call it so I'm naming it Safari burps. This is when Safari hold a months' worth of browsing history even though you have it set to one day. The only way to get rid of this is to clear all browsing history. and then make sure to clear all ~/Library/Cache and Saved Application State cache as well, then reboot. 

More concerning is a ticket that I have open with Cisco Umbrella. I presumed it was the Safari option - Check for fraudulent sites... but there appeared a 24x7 (even when Safari wasn't open) check to websitereview.corp.apple.com in my Cisco Umbrella DNS logs. 

Here is what Cisco said:

Thank you for contacting Cisco OpenDNS support.

We can find or access the domain websitereview.corp[.]apple[.]com and when we try we receive a message that says This site can’t be reachedCheck if there is a typo in [websitereview.corp.apple.com](http://websitereview.corp.apple.com/).
DNS_PROBE_FINISHED_NXDOMAIN

This could be a domain generated by some sort of application on one of the user's workstations that's "pinging" to see if there's a live internet connection. It could be an antivirus program, or any other application that does such checks.

This made me take a look at my activity logs... and this website was number one with over 7k request (14x the next website), even when I wasn't running Safari!!!! And I couldn't see this request in my flow logs. But once I did the clear browsing history/cache/saved application state (aka Safari Burp) this disappeared. I think I only caught this traffic because I cut DNS leaks and set all DNS traffic to Umbrella servers (even port 53). 

Have you seen Safari Burps? No one else seems to have these issues but me... and they are persistent across different machines and OSes.

[apple 24 hours (America_New_York).csv](https://github.com/usnistgov/macos_security/files/11266007/apple.24.hours.America_New_York.csv)
<img width="918" alt="Screenshot 2023-04-18 at 2 30 34 AM" src="https://user-images.githubusercontent.com/102542366/232895312-2c97f76d-e104-4ab3-966c-a6ab5fff38ab.png">
<img width="1009" alt="Screenshot 2023-04-18 at 2 32 25 AM" src="https://user-images.githubusercontent.com/102542366/232895315-ea8c38be-afa6-479b-85ca-7a7699527344.png">
<img width="913" alt="Screenshot 2023-04-18 at 2 31 08 AM" src="https://user-images.githubusercontent.com/102542366/232895318-ff143c0d-06c1-4099-90ee-9bc9f3d59526.png">
<img width="1423" alt="Screenshot 2023-04-17 at 10 25 14 PM" src="https://user-images.githubusercontent.com/102542366/232895320-b847458d-93d8-4310-8d29-07f7b28aa35f.png">

**Comment by ghost** [comment]
Cisco Umbrella says they have no way to triage this... since it's not something they can't prevent from their side. Any suggestions?

**Comment by ghost** [comment]
which means it has to be something local on the network I don't control at the Manhattan Bowery Lodge.

**Comment by ghost** [comment]
I got this CSRF error on Safari... when loading very plain Jane websites. I had to burp Safari on my iPhone but because of syncing I wonder if someone could use this as a backdoor across systems.  

![IMG_0194](https://github.com/usnistgov/macos_security/assets/102542366/968d585f-d559-4658-acb3-a53ad90b3b5e)

**Reply by ghost** [reply]
I had to burp safari again but it may be tied into Private Relay. Even though I've seen this same error for the last five Mac OS versions or so. Any suggestions?

**Reply by jamierrichardson** [reply]
Vaughn, this doesn't seem like an issue with the functioning of this project.  If this is an issue you have experienced with multiple releases of macOS, perhaps consider contacting  AppleCare support?

**Comment by ghost** [comment]
I cleared my cache and installed Adsorbs and Safari isn't congested anymore... for the last week or so. No need to burp it!  Someone's cookies can trigger Safari caching and keeping your browsing history beyond your settings.... is what I'm saying.

**Comment by ghost** [comment]
https://pi-hole.net

This is the basis for Adsorb.
